apps/website/src/pages/api/short-links/index.ts

```diff
@@ -0,0 +1,16 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { prisma } from "@/server/db/client";
+
+//API for short links middleware function
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  const body = JSON.parse(req.body);
+  const slug = body.slug;
+
+  const shortLink = await prisma.shortLinks.findFirst({
+    where: { slug: slug },
+  });
```

Comment:
```suggestion
  const body = JSON.parse(req.body);

  const shortLink = await prisma.shortLinks.findFirst({
    where: { slug: body.slug },
  });
```
The slug const feels redundant here (body in theory is too, but I can see logically it makes sense to keep it in a const)